crates/uv-cli/src/lib.rs

```diff
@@ -1382,6 +1382,14 @@ pub struct PipCompileArgs {
     #[arg(long, alias = "exclude", env = EnvVars::UV_EXCLUDE, value_delimiter = ' ', value_parser = parse_maybe_file_path)]
     pub excludes: Vec<Maybe<PathBuf>>,
 
+    /// Exclude the given package names from resolution.
```

Comment:
I think we need to copy the additional content from `--excludes` above.

> When a package is excluded, it will be omitted from the dependency list entirely and its own dependencies will be ignored during the resolution phase